docs/pages/dev/reference/API/org_unit_registry.md

```diff
@@ -0,0 +1,436 @@
+      "altitude": "Double - New altitude of the OrgUnit",
+      "accuracy": "Double - New accuracy of the OrgUnit"
+    },
+    "geo-shape": "String? - GeoJSON representation, may be null or omitted",
+    "forms": [ "Array of form objects? - may be null or omitted, cannot be empty",
+      {
+        "form_id": "id of the form",
+        "instance_id": "UUID of the instance"
+      }
+    ]
+  }
+}
+```
+
+## Possible responses
+
+### 201 - Created
+
+### 400 - Bad request
+
+- `app_id` was not provided
+- A not nullable field was null or omitted
+- `org_unit_id` is a new OrgUnit and the following fields were null or omitted: `parent_id`, `name`, `geo_point` and `org_unit_type_id`
+- A `String` field has an empty value
+- `parent_id` is not a valid OrgUnit
+- `status` was not one of the enum values.
+- `org_unit_type_id` is not a valid OrgUnitType
+- One of the `groups` id is not a valid Group
+- `forms` is empty
+- `form_id` is not a reference form for the OrgUnitType
+- `org_unit_type_id` is not a valid sub OrgUnitType for the given parent --> TO BE CONFIRMED
+
+### 401 - Unauthorized
+
+- No authentication token or an invalid one was provided
+
+### 403 - Forbidden
+
+- `org_unit_type_id` is not a valid OrgUnitType to be created for the given `parent_id`
+- `org_unit_type_id` is not a valid OrgUnitType to be moved to the given `parent_id`
+
+### 404 - Not found
+
+- `form_id` is not found
+- `instance_id` is not found
+- `parent_id` is not found
+- `org_unit_type_id` is not found
+
+
+
+
+# GET /api/orgunits/changes/
+
+## Permissions
+
+Org Unit admin?
+
+## Query parameters
+- page: Int (optional) - Current page (default: 1)
+- limit: Int (optional) - Number of entities returned by page (default: 20)
+- org_unit_id: Int (optional) - Id of the OrgUnit to which the changes apply (default: null)
+- org_unit_type_id: Int (optional) - Id of the OrgUnitType to filter on, either the current OrgUnitType or in the change (default: null)
+- status: Array<Enum<Status>> (optional) - One of `new`, `validated`, `rejected` to filter the requests (default: null)
+- parent_id: Int (optional) - Id of the parent OrgUnit to filter on, either the current parent or in the change (default: null)
+- groups: List of int, comma separated (optional) - Ids of the group to filter on, either the currents groups or in the change (default: null)
+- project: Int (optional) - Id of the project to filter on.
+
+## Possible responses
+
+### 200 - OK
+
+```json
+{
+  "count": "Long",
+  "has_next": "Boolean",
+  "has_previous": "Boolean",
+  "page": "Long",
+  "pages": "Long",
+  "limit": "Long",
+  "results": [
+    {
+      "id": "Int - id in the database, I know Martin wants it",
+      "uuid": "UUID - provided by the client when request was created",
+      "status": "Enum<Status> - one of `new`, `validated`, `rejected`",
+      "org_unit_name": "String - name of the OrgUnit",
+      "org_unit_type_name": "String - name of the current OrgUnitType",
```

Comment:
i would add org unit type id too here, or a small object sith id and name

Reply:
Good point, since it's possible to have multiple org unit types with the same name